.github/workflows/bump-swc.yml

```
@@ -0,0 +1,43 @@
+name: Bump up swc
+  on:
+    schedule:
+      # two times daily, at 12:00 and 06:00
+      - cron: '0 0 * * *'
+      - cron: '0 6 * * *'
+
+jobs:
+  upgrade-swc-core:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - uses: actions/cache@v3
+      with:
+        path: |
+          ~/.cargo/bin/
+        key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+
+    - name: Install Rust
+      uses: actions-rs/toolchain@v1
```

```suggestion
      uses: dtolnay/rust-toolchain@stable
```

`actions-rs` is abandoned unfortunately

we'll probably want to switch the other ones as well, created a ticket for it here: https://linear.app/vercel/issue/WEB-258/switch-actions-rstoolchain-github-action-to-dtolnayrust-toolchain

---

Cool, thanks for the heads up.